### ISSUE TITLE
Unresolvable dependency and BindingResolutionException on Laravel 5.4

### ISSUE THREAD

**Issue by gandra** [asvae/laravel-api-tester#35]
After installing package on laravel 5.4 `php artisan   route:list` fails:

```
$ php artisan   route:list

  [Illuminate\Contracts\Container\BindingResolutionException]
  Unresolvable dependency resolving [Parameter #2 [ <required> $path ]] in class Asvae\ApiTester\Storages\JsonStorage
```

My environment:
Windows 7
php 7.1
Laravel 5.4

**Comment by asvae** [comment]
Thanks.

This means that library essentially doesn't work on laravel 5.4. I'll take a look promtly.

**Comment by greabock** [comment]
laravel/framework#17556

**Comment by asvae** [comment]
Fixed in [2.0.6](https://github.com/asvae/laravel-api-tester/releases/tag/2.0.6).

**Comment by asvae** [comment]
@gandra please reopen if I missed someting.
